includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -988,6 +997,34 @@ public function schedule_stripe_billing_migration( WP_REST_Request $request = nu
 		}
 	}
 
+	/**
+	 * Request a specific capability.
+	 *
+	 * @param WP_REST_Request $request The request object. Optional. If passed, the function will return a REST response.
+	 *
+	 * @return WP_REST_Response|WP_Error The response object, if this is a REST request.
+	 */
+	public function request_capability( WP_REST_Request $request = null ) {
+		$id                      = $request->get_param( 'id' );
+		$capability_key_map      = $this->wcpay_gateway->get_payment_method_capability_key_map();
+		$payment_method_statuses = $this->wcpay_gateway->get_upe_enabled_payment_method_statuses();
+		$cache_needs_refresh     = false;
+		$stripe_key              = $capability_key_map[ $id ] ?? null;
+
+		if ( array_key_exists( $stripe_key, $payment_method_statuses ) ) {
+			if ( 'unrequested' === $payment_method_statuses[ $stripe_key ]['status'] ) {
+				$request_result      = $this->api_client->request_capability( $stripe_key, true );
+				$cache_needs_refresh = $cache_needs_refresh || 'unrequested' !== $request_result['status'];
+			}
+		}
+
+		if ( $cache_needs_refresh ) {
```

Comment:
we can add this block under `if ( 'unrequested' === $payment_method_statuses[ $stripe_key ]['status'] ) {`, and remove `$cache_needs_refresh     = false;` line

Reply:
Good suggestion, updated!